packages/effect/src/Tuple.ts

```diff
@@ -34,7 +34,7 @@ export interface TupleTypeLambda extends TypeLambda {
 export const make = <A extends ReadonlyArray<any>>(...elements: A): A => elements
 
 /**
- * Return the first element of a tuple.
+ * Return the first element of a 2-tuple.
```

Comment:
```suggestion
 * Return the first element from a tuple with two elements.
```

I think this reads better :)